hil-test/tests/gpio.rs

```diff
@@ -1,8 +1,8 @@
 //! GPIO Test
 //!
 //! Folowing pins are used:
-//! GPIO2 / GPIO9 (esp32s2 and esp32s3)
-//! GPIO3 / GPIO10 (esp32s2 and esp32s3)
+//! GPIO2 / GPIO9 / GPIO26  (esp32s2 / esp32s3 / esp32)
+//! GPIO3 / GPIO10 / GPIO26 (esp32s2 / esp32s3 / esp32)
```

Comment:
I think the information here is not correct, AFAIK:
- 2-3 are used for c2,c3,h2
- 9-10 in s2 and s3
- 26-27 in esp32 (there is also a typo and says 26-26 in code)